backend/docs/API_Specifications/asset-registry-1.0.0.yaml

```
@@ -0,0 +1,710 @@
+          maxLength: 64
+        assetDetails:
+          type: object
+          description: Custom key-value pair details of the asset specific to asset type. These can be defined in the master schema per asset type. The schema should be used to validate the payload.
+          additionalProperties: true
+        warrantyStartDate:
+          type: string
+          format: datetime
+          description: Start date of the warranty in ISO 8601 format with Z offset
+        warrantyDuration:
+          type: string
+          description: Duration of the warranty in ISO 8601 format (e.g., P1Y2M) 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)